docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -0,0 +1,342 @@
+generally has the form `[[SOURCE-VOLUME|HOST-DIR:]CONTAINER-DIR[:OPTIONS]]`.
+
+As a special case, if `SOURCE-VOLUME` ends with `.volume`, a Podman named volume called
+`systemd-$name` will be used as the source, and the generated systemd service will contain
+a dependency on the `$name-volume.service`. Such a volume can be automatically be lazily
+created by using a `$name.volume` quadlet file.
+
+This key can be listed multiple  times.
+
+#### `ExposeHostPort=`
+
+Exposes a port, or a range of ports (e.g. `50-59`) from the host to the container. Equivalent
+to the Podman `--expose` option.
+
+This key can be listed multiple  times.
+
+#### `PublishPort=`
+
+Exposes a port, or a range of ports (e.g. `50-59`) from the container to the host. Equivalent
+to the Podman `--publish` option. The format is similar to the Podman options, which is of
+the form   `ip:hostPort:containerPort`, `ip::containerPort`, `hostPort:containerPort` or
+`containerPort`, where the number of host and container ports must be the same (in case
+of a range).
+
+If the IP is set to 0.0.0.0 or not set at all, the port will be bound on all IPv4 addresses on
+the host, use [::] for IPv6.
+
+Note that not listing a host port means that Podman will automatically select one, and it
+may be different for each invocation of service. This makes that a less useful option.  The
+allocated port can be found with the `podman port` command.
+
+This key can be listed multiple  times.
+
+#### `PodmanArgs=`
+
+This key contains a list of arguments passed directly to the end of the `podman run` command
+in the generated file (right before the image name in the command line). It can be used to
+access Podman features otherwise unsupported by the generator. Since the generator is unaware
+of what these options to it can cause unexpected interactions, so it is not recommended to use
+this option.
+
+This key can be listed multiple  times.
+
+#### `Label=`
+
+Set one or more OCI labels on the container. The format is a list of `key=value` items,
+similar to `Environment`.
+
+This key can be listed multiple  times.
+
+#### `Annotation=`
+
+Set one or more OCI annotations on the container. The format is a list of `key=value` items,
+similar to `Environment`.
+
+This key can be listed multiple  times.
+
+### Volume units
+
+Volume files are named with a `.volume` extension and contain a section `[Volume]` describing the
+named Podman volume. The generated service is a one-time command that ensures that the volume
+exists on the host, creating it if needed.
+
+For a volume file named `$NAME.volume`, the generated Podman volume will be called `systemd-$NAME`,
+and the generated service file `$NAME-volume.service`.
+
+Using volume units allows containers to depend on volumes being automatically pre-created. This is
+particularly interesting when using special options to control volume creation, as Podman will
+otherwise create volumes with the default options.
+
+Supported keys in `Volume` group are:
```

Comment:
`Volume` group or `Volume1 section?  I think section, at least to be consistent with your first line here.